tests/pyccel/scripts/expressions.py

```diff
@@ -14,15 +16,15 @@
 a10 = 1.*10j
 
 b1 = 1-1-2-4-5
-b2 = 1-1-2j-4.-5
-b3 = 1-1-2j-5
+# b2 = 1-1-2j-4.-5
+# b3 = 1-1-2j-5
```

Comment:
Please mark failing tests as `xfail` rather than removing them. Removing the test makes it easier for us to forget that this is broken.
Worse, if this is only broken for one language then it can be broken for another language in the future without us noticing

Reply:
If I mark this as `xfail` then other expressions will not be tested.
and in the future, someone may break these passed tests without knowing about them, instead of having a few expressions fails, so the problem gets bigger, 
maybe separate them from `expression.py` then `xfail` them in c.
what do you think?

Reply:
That's true but I think most (all?) of the basic things have tests in the `epyccel` folder. If everything is tested there then an `xfail` is sufficient. If not then I agree the best thing would be to separate them or move these tests to `epyccel` so the tests can be more granular

Reply:
done.